### ISSUE TITLE
Suggest using the property/event vs. getting the accessor by name, when type is unknown

### ISSUE THREAD

**Issue by jnm2** [DotNetAnalyzers/ReflectionAnalyzers#72]
Can we make #19 apply on a best-effort basis when types aren't known at analysis time by recognizing the common patterns `get_`, `set_`, `add_`, `remove_` and `raise_`?

**Comment by JohanLarsson** [comment]
This is a simple fix, question is if we should give it a separate id so it becomes configurable?

**Comment by jnm2** [comment]
I probably wouldn't. Can you split it out later upon request?

**Comment by JohanLarsson** [comment]
Yes we can split later, slight breaking change if someone has disabled it because it was annoying but then we are early alpha.
